### ISSUE TITLE
Elementor tabs & accordion are not accessible

### ISSUE THREAD

**Issue by itroww** [elementor/elementor#3576]
## Description

Elementor tabs & accordion are not accessible for Special Needs.

## Steps to reproduce

by using "Tab" in the keyboard the tabs & accordion elements are open one after each other but I cant reach the "links" inside them.
It should open the elements, and let me navigate the links inside them and so on.

## To fix it
Its looks like every link inside the elelment should get the same "tabindex" as the  (tabs & accordion) element title.

Thanks

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.1.3

Feel free to update

Thanks!
